src/components/Header.js

```diff
@@ -1,23 +1,77 @@
 // @flow
 import React from "react";
+import { StatusBar, StyleSheet, View } from "react-native";
 import type { Node } from "react";
-import { StatusBar } from "react-native";
 import SafeAreaView from "react-native-safe-area-view";
+import ContentPadding from "./ContentPadding";
+import IconButton from "./IconButton";
+import Text from "./Text";
+import chevronLeftWhite from "../../assets/images/chevron-left-white.png";
+import { lightNavyBlueColor } from "../constants/colors";
 
 type Props = {
-  backgroundColor: string,
-  children: Node
+  onBack?: () => void,
+  title?: string,
+  rightElement?: Node
 };
 
-const Header = ({ backgroundColor, children }: Props) => (
-  <SafeAreaView accessibilityTraits={["header"]} style={{ backgroundColor }}>
-    <StatusBar
-      barStyle="light-content"
-      animated
-      backgroundColor={backgroundColor}
-    />
-    {children}
-  </SafeAreaView>
+const renderBackButton = onBack => (
+  <IconButton
+    accessibilityLabel="Back"
```

Comment:
This might be a good opportunity to move this `"Back"` string in our `constants/text.js` file.